src/server/scripts/Northrend/CrusadersColiseum/TrialOfTheChampion/boss_argent_challenge.cpp

```diff
@@ -1,5 +1,5 @@
 /*
- * This file is part of the TrinityCore Project. See AUTHORS file for Copyright information
+ * Copyright (C) 2008-2018 TrinityCore <https://www.trinitycore.org/>
```

Comment:
Looks like you are off to a better start with this PR compared to the previous one. Even so, I recommend integrating the current TC copyright information in these script files:
```suggestion
 * This file is part of the TrinityCore Project. See AUTHORS file for Copyright information
```